src/Network/Network/ChangeLog.md

```diff
@@ -33,6 +33,8 @@
         - `Get-AzApplicationGatewayFirewallPolicy`
         - `Set-AzApplicationGatewayFirewallPolicy`
         - `New-AzApplicationGatewayFirewallPolicy`
+* Added new cmdlet for List NetworkSecurityPerimeter ServiceTags
```

Comment:
please help move this line to line 22 right under `Upcoming Release`, this will help better resolve conflicts, thanks